documentation/smg/02_development_environment.md

```diff
@@ -1 +1,102 @@
 # Development Environment
+
+### Scope
```

Comment:
Since these are second level headers, should we have `##` instead of `###` here?

Reply:
As a personal preference, I find that for the top 2 levels, the font is too large. So preferring to start at level 3 - no-one has picked up on it before. But happy to conform to usage.